src/main/java/Mediell/Task.java

```diff
@@ -0,0 +1,61 @@
+package Mediell;
+
+import java.util.Objects;
+
+/** Represents a Task. */
+public class Task {
+    private String taskName;
+    private boolean completed;
+
+    public Task(String taskName) {
+        this.taskName = taskName;
+        completed = false;
+    }
+    
+    public boolean find(String keyword) {
+        return taskName.contains(keyword);
+    }
+
+    /**
+     * Marks the task as completed.
+     */
+    public void markAsCompleted() {
+        completed = true;
+    }
+
+    /**
+     * Marks the task as uncompleted.
+     */
+    public void markAsUncompleted() {
+        completed = false;
+    }
+
+    @Override
+    public String toString() {
+        String status = "";
+        if (completed) {
+            status = "[X] ";
```

Comment:
Could adding a whitespace cause formatting issues down the road? Why not add it in the class where the final string is consolidated?